README.md

```diff
@@ -20,26 +20,31 @@ Or install it yourself as:
 
 ## Usage
 
-### Sinatra instructions
-#### App setup
-* Add to Gemfile as well as listen 3.1.1
-* Require in server file (i.e. config.ru)
```

Comment:
I didn't have to modify config.ru at all, since I ran the `opal-hot-reloader` script in a separate terminal -- the config.ru then doesn't care or know about the hot reloader running on the separate port. Does it know how to run when I start up sinatra otherwise?